app/grants/views.py

```diff
@@ -460,8 +465,10 @@ def get_grants(request):
     round_num = request.GET.get('round_num', None)
     sub_round_slug = request.GET.get('sub_round_slug', '')
     customer_name = request.GET.get('customer_name', '')
-    sort = request.GET.get('sort_option', 'weighted_shuffle')
+    sort = request.GET.get('sort_option', None)
+    tenants = request.GET.get('tenants', '')
```

Comment:
is missing the regions filter

Reply:
@octavioamu added ! You can now filter using 
http://localhost:8000/grants/cards_info?idle=true&grant_regions=oceania,africa,north_america